vue.config.js

```diff
@@ -9,6 +9,12 @@ const commitHash = require('child_process')
 
 module.exports = {
   lintOnSave: false,
+  pluginOptions: {
+    ssr: {
+      favicon: './dist/assets/favicon.ico',
```

Comment:
The "Cannot GET /" error was because of this https://github.com/Akryum/vue-cli-plugin-ssr/blob/e9d0c4fc448e11805edde294d2cb9654c1daa3ce/lib/app.js#L94
is was caused by a wrong path to favicon.